### ISSUE TITLE
better way to measure # of images with missing alt text

### ISSUE THREAD

**Issue by MoralCode** [CampusPulse/access-directory#38]
heres some queries, this should maybe become a /stats endpoint or something

```sql
select * from  images i  where i.alttext is null

--select * from join access_point_image_relation apir on apir.image_id = i.id


select * from access_point_image_relation apir 
```
